src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.tsx

```diff
@@ -48,24 +56,38 @@ const UserTokenActivationButton = ({
       colonyAddress={colonyAddress}
       isPendingBalanceZero={isPendingBalanceZero}
     >
-      {({ toggle, ref }) => (
+      {({ isOpen, toggle, ref }) => (
         <>
           <button
             type="button"
             className={styles.tokens}
             onClick={toggle}
             ref={ref}
           >
-            <span
-              className={`${styles.dot} ${
-                (inactiveBalance.gt(0) || totalBalance.isZero()) &&
-                styles.dotInactive
-              }`}
-            />
-            <Numeral
-              suffix={` ${nativeToken?.symbol} `}
-              value={formattedTotalBalance}
-            />
+            <Tooltip
+              appearance={{ theme: 'dark', size: 'medium' }}
+              placement="bottom"
+              trigger={!isOpen ? 'hover' : 'disabled'}
```

Comment:
Why not `isOpen ? 'disabled' : 'hover'` ?

Reply:
You say 'pOtatOes' i say 'pAhtAtoes'.... 

No real reason, either works. 
Although i tend to think of the 1st option as the primary flow, and 2nd as well..... secondary. So then it comes down to the meaning/logic of the flag vs meaning/logic of the action, and i was focused on the 'hover'.